### ISSUE TITLE
Discontinuity

### ISSUE THREAD

**Issue by BestGiter** [TodePond/Arroost#304]
In the video for Arroost, it says like the arrow of infinity can go into the future and past by powering it and stuff, but it doesn’t work, and in it, is the same universe

**Comment by TodePond** [comment]
yes it is an unreleased feature. there is a branch sitting around somewhere for it !

**Comment by BestGiter** [comment]
what branch, what is a branch

**Comment by BestGiter** [comment]
> yes it is an unreleased feature. there is a branch sitting around somewhere for it !

where is the branch, does it exist
